docs/neo3_guides/compiler_devpack/compilation.md

```diff
@@ -41,12 +13,12 @@ as a Gradle task. Add the following few lines to your `build.gradle`.
 // Add the neow3j gradle plugin to the plugins section.
 plugins {
     id 'java'
-    id 'io.neow3j.gradle-plugin' version "3.3.0"
+    id 'io.neow3j.gradle-plugin' version "3.4.0"
 }
 
 // Add the devpack to the dependencies.
 dependencies {
-    compile("io.neow3j:devpack:3.3.0")
+    compile("io.neow3j:devpack:3.4.0")
```

Comment:
the same comment from above applied here.

Reply:
ok, i'll change it to 3.+

Reply:
I know that 3.+ works here. 😄